Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -172,13 +193,15 @@ class PushNotificationIOS {
       listener =  PushNotificationEmitter.addListener(
         DEVICE_NOTIF_EVENT,
         (notifData) => {
+          notifData.isRemote = true;
```

Comment:
Hmmm well looking at Github's master I see this: https://github.com/facebook/react-native/blob/master/Libraries/PushNotificationIOS/PushNotificationIOS.js#L175-L177 which does NOT have isRemote. Maybe you're seeing something I'm not? Possibly something that's internal to Facebook members only? I don't know. I just updated with master and this is the diff though.